### ISSUE TITLE
Menu management failure

### ISSUE THREAD

**Issue by pongraczi** [crocodic-studio/crudbooster#987]
- Operating system (name, version and architecture)
Debian 9.2 amd64
- Web server (name, version and architecture)
Apache 
- CRUDBooster version
Latest on 10th of Dec., 2017
- Laravel version
5.5
- Database
Postgresql 9.6
- What is the problem?
Menu management dies.
- How should it happen? What was the expected behavior?
Click on the Menu management anytime after a fresh install.
- Tell how to reproduce the problem step by step
Use Postgresql 9.6 as database.
- If possible, take screenshots or send snippets of code
![kijeloles_621](https://user-images.githubusercontent.com/5891983/33810547-3aa48dc0-de06-11e7-8fff-6d6196e3572c.png)

- Possible reason
In postgresql, 0 is not false, 1 is not true. You cannot compare boolean to integer. I guess, the database created by crudbooster should be more specific and use smallint/integer instead of boolean or use true/false if boolean should be keep. I prefer to change to true/false from1/0.
In other words, if something is boolean, do not use integer (0,1) as value, even other DB allows that.

**Comment by pongraczi** [comment]
This one solved the issue, at least it can display the menu management instead of error page.
#317 
in vendor/crocodicstudio/crudbooster/src/controllers/MenusController.php Line no 158 replace "is_protected = FALSE"
